src/main/java/spoon/support/compiler/jdt/JDTCommentBuilder.java

```diff
@@ -373,6 +375,14 @@ public void visitCtAnonymousExecutable(CtAnonymousExecutable e) {
 			public <T> void visitCtParameter(CtParameter<T> e) {
 				e.addComment(comment);
 			}
+
+			@Override
+			public void visitCtCatch(CtCatch e) {
+				if (comment.getPosition().getLine() <= e.getPosition().getLine()) {
+					e.addComment(comment);
+					return;
+				}
```

Comment:
This super call is directly implemented in the `scan` override, do we really need it here? It is missing in all the other overriding `visitCtXX` methods...